engine/lib/navigation.php

```diff
@@ -241,17 +241,20 @@ function elgg_register_title_button($handler = null, $name = 'add', $entity_type
  *
  * See elgg_get_breadcrumbs() and the navigation/breadcrumbs view.
  *
- * @param string $title The title to display. During rendering this is HTML encoded.
- * @param string $link  Optional. The link for the title. During rendering links are
- *                      normalized via elgg_normalize_url().
+ * @param string $text The title to display. During rendering this is HTML encoded.
+ * @param string $href Optional. The href for the title. During rendering links are
+ *                     normalized via elgg_normalize_url().
  *
  * @return void
  * @since 1.8.0
  * @see elgg_get_breadcrumbs
  */
-function elgg_push_breadcrumb($title, $link = null) {
+function elgg_push_breadcrumb($text, $href = null) {
```

Comment:
I can live without it. I have thrown out breadcrumbs out of my latest theme, because the navigation is a lot more logical and you can figure out where you are based on header and tabs.